src/EA.Weee.Integration.Tests/Handlers/GetEvidenceNoteForSchemeHandlerIntegrationTests.cs

```diff
@@ -0,0 +1,197 @@
+
+                var categories = new List<NoteTonnage>()
+                {
+                    new NoteTonnage(WeeeCategory.AutomaticDispensers, 2, 1),
+                    new NoteTonnage(WeeeCategory.ConsumerEquipment, null, null),
+                    new NoteTonnage(WeeeCategory.GasDischargeLampsAndLedLightSources, 0, 0)
+                };
+
+                note = EvidenceNoteDbSetup.Init().WithTonnages(categories)
+                    .WithOrganisation(organisation.Id)
+                    .With(n =>
+                    {
+                        n.UpdateStatus(NoteStatus.Submitted, UserId.ToString());
+                    })
+                    .Create();
+                
+                request = new GetEvidenceNoteRequest(note.Id);
+            };
+
+            private readonly Because of = () =>
+            {
+                result = Task.Run(async () => await handler.HandleAsync(request)).Result;
+
+                note = Query.GetEvidenceNoteById(note.Id);
+            };
+
+            private readonly It shouldHaveCreatedEvidenceNote = () =>
+            {
+                note.Should().NotBeNull();
+            };
+
+            private readonly It shouldHaveCreatedTheEvidenceNoteWithExpectedPropertyValues = () =>
+            {
+                ShouldMapToNote();
+                note.Status.Should().Be(NoteStatus.Submitted);
+            };
+        }
+
+        [Component]
+        public class WhenIGetANoteWhereNoteDoesNotExist : GetEvidenceNoteForSchemeHandlerIntegrationTestBase
+        {
+            private readonly Establish context = () =>
+            {
+                LocalSetup();
+
+                request = new GetEvidenceNoteRequest(Guid.NewGuid());
+            };
+
+            private readonly Because of = () =>
+            {
+                CatchExceptionAsync(() => handler.HandleAsync(request));
+            };
+
+            private readonly It shouldHaveCaughtArgumentNullException = ShouldThrowException<ArgumentNullException>;
+        }
+
+        [Component]
+        public class WhenIGetANotesWhereUserIsNotAuthorised : GetEvidenceNoteForSchemeHandlerIntegrationTestBase
+        {
+            private readonly Establish context = () =>
+            {
+                LocalSetup();
+
+                note = EvidenceNoteDbSetup.Init().Create();
+
+                request = new GetEvidenceNoteRequest(note.Id);
+            };
+
+            private readonly Because of = () =>
+            {
+                CatchExceptionAsync(() => handler.HandleAsync(request));
+            };
+
+            private readonly It shouldHaveCaughtArgumentException = ShouldThrowException<SecurityException>;
+        }
+
+        public class GetEvidenceNoteForSchemeHandlerIntegrationTestBase : WeeeContextSpecification
+        {
+            protected static IRequestHandler<GetEvidenceNoteRequest, EvidenceNoteData> handler;
```

Comment:
This should be new GetEvidenceNoteBySchemeRequest

Reply:
line has been fixed
protected static IRequestHandler<GetEvidenceNoteForSchemeRequest, EvidenceNoteData> handler;